### ISSUE TITLE
Feature Request: bedtools getfasta show name _and_ chromosome position

### ISSUE THREAD

**Issue by tomck** [arq5x/bedtools2#406]
I desired this and found someone else [wanted the same thing](http://bedtools.readthedocs.io/en/latest/content/tools/getfasta.html#comment-2659563764), so I'm not alone.

Presently, using the -name feature removes the region, but if one is using cufflinks generated XLOCs, it can be very helpful to have the name and the region (for BLAST results, for example).

**Comment by arq5x** [comment]
Agreed that this would be useful - would you be interested in submitting a pull request?

**Comment by tomck** [comment]
Not proficient enough to do this myself, sorry.

**Comment by arq5x** [comment]
Understood. I will try to tackle this when I get some free time.

**Comment by rishavray** [comment]
I have added the requested feature locally in my system and I would be issuing a pull request shortly.

**Comment by ifiddes** [comment]
Please see issue #433. This update breaks tools used by other people. I can try to find time to submit a pull request, but it seems that all three options should be possible - name only, position only, or both.

**Comment by arq5x** [comment]
I completely agree. This will be fixed soon. Apologies for the inconvenience, I didn't appreciate the broader impact when accepting the PR.
